pkg/instancetype/instancetype.go

```diff
@@ -964,7 +969,7 @@ func applyCPU(field *k8sfield.Path, instancetypeSpec *instancetypev1beta1.Virtua
 	switch GetPreferredTopology(preferenceSpec) {
 	case instancetypev1beta1.PreferCores:
 		vmiSpec.Domain.CPU.Cores = instancetypeSpec.CPU.Guest
-	case instancetypev1beta1.PreferSockets:
+	case instancetypev1beta1.PreferSockets, instancetypev1beta1.PreferAny:
```

Comment:
Need test coverage for this.

Reply:
This should be covered in this test - https://github.com/kubevirt/kubevirt/pull/10547/files#diff-218f2952941d0ec349e5274a0cc9d5399f23c596afad577f824d323b81cd7422R1033